app/models/hubstats/pull_request.rb

```diff
@@ -11,7 +11,7 @@ class PullRequest < ActiveRecord::Base
     scope :belonging_to_users, lambda {|user_id| where(user_id: user_id.split(',')) if user_id}
     scope :group, lambda {|group| group_by(:repo_id) if group }
     scope :with_state, lambda {|state| (where(state: state) unless state == 'all') if state}
-    scope :with_label, lambda { |label_name| ({ :joins => :labels, :conditions => {:hubstats_labels => {name: label_name.split(',')} } }) if label_name }
+    scope :with_label, lambda {|label_name| ({ :joins => :labels, :conditions => {:hubstats_labels => {name: label_name.split(',')} } }) if label_name }
```

Comment:
Taking the whitespace out is more consistent with the other scopes in this file...